Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.py

```diff
@@ -199,6 +199,13 @@ def main() -> None:
             result = test_module_command(client)
             return_results(result)
 
+        elif demisto.command() == 'gra-validate-api':
```

Comment:
I believe this validation logic, shall be put under the 'test_module' code, allowing it to be executed when pressing the 'Test' button within an integration instance configuration.

Reply:
Ok, I see it's already there, why would we need to validate the API key within a command if it's already done once configuring the instance?